hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -8479,13 +8481,35 @@ public void closeRegionOperation(Operation operation) throws IOException {
       stores.values().forEach(HStore::postSnapshotOperation);
     }
     Thread thisThread = Thread.currentThread();
-    regionLockHolders.remove(thisThread);
+    removeRPCThreadFromMap(thisThread);
     lock.readLock().unlock();
     if (coprocessorHost != null) {
       coprocessorHost.postCloseRegionOperation(operation);
     }
   }
 
+  private boolean isOperationInterruptible(Operation op) throws IOException {
+    boolean isInterruptableOp = false;
+    switch (op) {
```

Comment:
I am still trying to find why other operations are not interruptible. @Apache9 Do you happen to know the reason?